src/Sulu/Component/Content/ContentTypeInterface.php

```diff
@@ -95,31 +95,6 @@ public function remove(
         $segmentKey
     );
 
-    /**
-     * TODO: Remove this before 2.0 release.
-     *
-     * Returns a template to render a form.
-     *
-     * @return string
-     */
-    public function getTemplate();
-
-    /**
-     * Returns default parameters.
-     *
-     * @param PropertyInterface|null $property
-     *
-     * @return array
-     */
-    public function getDefaultParams(PropertyInterface $property = null);
-
-    /**
-     * returns default value of content type.
-     *
-     * @return mixed
-     */
-    public function getDefaultValue();
```

Comment:
What exactly is this doing? Are we sure that we don't need default values in the future? :thinking:

Reply:
Think a field type can have a default value but not a content type. A content type should just say how to save the given value to PHPCR and  how to load view and content data. /cc @sulu/core-team